cloud/shared/bin/lib/terraform.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import os
+import shutil
+
+from cloud.shared.bin.lib import civiform_mode
+
+
+def perform_apply():
+    '''Generates terraform variable files and runs terraform init and apply.'''
```

Comment:
The original version claims that it initializes a storage bucket as well. Was that comment just incorrect or do we need to also move it here?

Reply:
we're setting it up as part of setup for aws, and should do the same for azure. I don't think it should be in this script at all. Added a comment so we don't forget.